onnx/backend/test/case/node/softmaxcrossentropy.py

```diff
@@ -178,3 +180,27 @@ def export_softmaxcrossentropy_mean_weights():  # type: () -> None
 
         # Check results
         expect(node, inputs=[x, labels, weights], outputs=[sce], name='test_softmax_cross_entropy_mean_weight')
+
+    @staticmethod
+    def export_softmaxcrossentropy_mean_weights_ignore_index():  # type: () -> None
```

Comment:
can we add a case with no weights and ignore_index specified?

Reply:
We spoke offline, now that ignore_index is not an input so this is not needed.